securedrop/tests/test_journalist.py

```diff
@@ -459,6 +460,24 @@ def test_admin_add_user_without_username(self):
                                           is_admin=False))
         self.assertIn('Missing username', resp.data)
 
+    @patch('journalist.app.logger.error')
+    @patch('journalist.Journalist')
+    def test_admin_add_user_integrity_error(self,
```

Comment:
While it's not uniformly implemented, a common pattern that we try to use in our unit tests is:

```
setup block
empty line
action(s) we're testing
empty line
assertion(s)
```

So inserting a line before and after your assignment of `resp` (that assignment being the "action(s) we're testing") would bring this code into line with that style, which would be preferable. The reason for doing this is pretty simple: many including myself (and maybe you'll agree) find sticking to this pattern to aide readability and clarity of test purpose. Sometimes you'll see

```
setup block
empty line
action(s) we're testing
empty line
assertion(s)
empty line
more action(s) we're testing
empty line
more assertion(s)
```

or other variations. For more complicated tests, just go with whatever seems most clear to you.